lib/Support/ErrorHandling.cpp

```diff
@@ -90,16 +91,22 @@ void llvm::report_fatal_error(const Twine &Reason, bool GenCrashDiag) {
 #ifndef LLVM_ON_WIN32 // HLSL Change - unwind if necessary, but don't terminate the process
   if (handler) {
     handler(handlerData, Reason.str(), GenCrashDiag);
-  } else {
+  } else
+#endif
```

Comment:
The effect of this plus the deleted code from the LLVM_ON_WIN32 branch below is that handler() is no longer called for win32 when it was before.  Is that intentional?